src/nextjournal/clerk/viewer.cljc

```diff
@@ -1889,6 +1899,7 @@
                                                                    (nextjournal.clerk.render/inspect-presented opts form)]
                                                                   [:div.pt-2.px-4.border-l-2.border-transparent
                                                                    (nextjournal.clerk.render/inspect-presented opts val)]])})
+                       (update-in [:nextjournal/value :val] maybe-wrap-var-from-def (get-in wrapped-value [:nextjournal/value :form]))
```

Comment:
Does `clerk/comment` need the same treatment?

Reply:
`clerk/comment` expands to a fragment, so it's covered by handling the latter.  I'll add a test.